en/index.md

```diff
@@ -523,6 +523,26 @@ Lastly, these Latin words and phrases are widespread in academia and research. T
 - stet
 
 
+<!-- RULE
+#26 Avoid orphan demonstrative pronouns
+-->
+
+## Lone "this", "these", "those", "that"
+
+A _demonstrative pronoun_ is a word that refers to a thing previously discussed, and are the familiar words "this", "these", "those" and "that". They can work both in isolation ("That's great!") and paired with the thing they refer to ("This apple is sweet.").
+
+Isolated demonstrative pronouns can improve the flow of sentences when there's no possibility of ambiguity:
+
+- Create two new instances. These will operate as your client and server.
+
+However, if there's a large distance between the pronoun and the thing you're referring to, or the previous sentence contains multiple ideas, the reader may confuse your meaning. If there's a chance of ambiguity, put the target noun immediately after a demonstrative pronoun:
```

Comment:
> if there's a large distance between the pronoun and the thing you're referring to

@edibotopic Do you think I should create an example for this?

Reply:
I think the examples you have in this current version are fine.

After the "create two new instances" example, however, you could add a line to the effect:

> Here, it is clear that "these" refers to two instances, where one is the client and one is the server.

Reply:
06c9bb481a117e9b09a6e12deb8123dfc94040c2